packages/cdk8s-plus/package.json

```diff
@@ -65,14 +66,72 @@
   ],
   "keywords": [
     "cdk",
-    "kubernetes",
+    "constructs",
     "k8s",
-    "constructs"
+    "kubernetes"
   ],
   "engines": {
     "node": ">= 10.17.0"
   },
+  "main": "lib/index.js",
   "license": "Apache-2.0",
+  "start": {
```

Comment:
Just out of curiosity - what's this?

Reply:
It's the metadata for `projen start` (added in the projen upgrade).